tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -92,4 +92,21 @@ Docker run immediate exit
     Should Not Contain  ${output}  Error
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run busybox
     Should Be Equal As Integers  ${rc}  0
-    Should Be Empty  ${output}
\ No newline at end of file
+    Should Be Empty  ${output}
+
+Docker run verify container start and stop time
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${cmdStart}=  Run  date +%s
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name startStop busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Empty  ${output}
+    ${rc}  ${containerStart}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect -f '{{.State.StartedAt}}' startStop | xargs date +%s -d
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${containerStop}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect -f '{{.State.FinishedAt}}' startStop | xargs date +%s -d
+    Should Be Equal As Integers  ${rc}  0
+    ${startStatus}=  Run Keyword And Return Status  Should Be True  ${cmdStart} <= ${containerStart}
+    Run Keyword Unless  ${startStatus}  Fail  container start time before command start
+    ${stopStatus}=  Run Keyword And Return Status  Should Be True  ${cmdStart} < ${containerStop}
```

Comment:
I'm changing this to containerStart < containerStop, but again I'm not checking for any specific value but just that the value makes sense.  The reason we created this test was due to NTP issue in host configuration that resulted in start times that would be before the command start, etc...or to ensure that's the only time that happens.

You say to add an additional check to verify accuracy...I'd ask that you define your notion of accurate.